docs/helm-broker/08-01-fetch-remote-addons.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Fetch addons from remote servers
+type: Tutorials
+---
+
+By default, the Helm Broker fetches addons listed in the `index.yaml` file from the `bundles` repository [release](https://github.com/kyma-project/bundles/releases). This tutorial shows how to configure the Helm Broker to fetch cluster-wide and Namespace-scoped addon definitions from other remote HTTPS servers.
+
+## Steps
+
+Follow these steps to configure the Helm Broker to fetch addon definitions from other remote HTTPS servers.
+
+<div tabs>
+  <details>
+  <summary>
+  Cluster-wide addons
+  </summary>
+
+  1. [Create a repository](#details-create-addons-repository) with your addons. To complete this tutorial step by step, use the existing [bundles](https://github.com/kyma-project/bundles/tree/master/bundles) repository.
```

Comment:
point here how to create a repo: https://github.com/kyma-project/bundles/blob/master/docs/getting-started.md